src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/pipeline/KendraRankingResponseProcessor.java

```diff
@@ -156,7 +159,9 @@ public KendraRankingResponseProcessor create(
                 Map<String, Processor.Factory<SearchResponseProcessor>> processorFactories,
                 String tag,
                 String description,
-                Map<String, Object> config
+                boolean IgnoreFailure,
```

Comment:
Nit comment, should this be capitalized? Think camelcase would mean it's `ignoreFailure` like the other lines.

Reply:
Thank you!!! changed to lowercase